bin/poeditor-pull.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash                                                                                                                                                                                     
+CURL='/usr/bin/curl'
+POEDITOR_EXPORT_URL="https://api.poeditor.com/v2/projects/export"
+API_TOKEN="$POEDITOR_API_TOKEN" # may be set as environment variable
```

Comment:
I think you should add something like this to ensure the API_TOKEN has a value

`[[ -z "$API_TOKEN" ]] && { echo "No API_TOKEN available!" ; exit 1; }`